docs/index.html

```diff
@@ -528,23 +528,57 @@ <h2>Terminology</h2>
         </dd>
         <dt>URI</dt>
         <dd>An identifier as defined by [[RFC3986]].</dd>
-        <dt>Workflow Definition</dt>
-        <dd>
+      </dl>
+
+      <section>
+        <h3>Extensions to Standard</h3>
+
+        <section>
+          <h4>Verifiable Credential</h4>
+
+          <p>The object MUST conform to the basic requirements section of the <a href="https://www.w3.org/TR/vc-data-model/#basic-concepts">W3C VC Data Model</a>.</p>
+
+          <p>The <code>@context</code> MUST contain <code>"https://w3id.org/traceability/v1"</code>.</p>
+
+          <p>The object MUST have an <code>id</code> property, and its value MUST be a valid IRI (URI, URN).</p>
+        </section>
+
+        <section>
+          <h4>Verifiable Presentation</h4>
+
+          <p>The object MUST conform to the basic requirements section of the <a href="https://www.w3.org/TR/vc-data-model/#basic-concepts">W3C VC Data Model</a>.</p>
+
+          <p>The <code>@context</code> MUST contain <code>"https://w3id.org/traceability/v1"</code>.</p>
+
+          <p>The object MUST have an <code>id</code> property, and its value MUST be a valid IRI (URI, URN).</p>
+        </section>
+
+        <section>
+          <h4>Workflow Definition</h4>
           <p>
             The sequence of industrial, administrative, or other processes
             through which a piece of work passes from initiation to completion.
           </p>
+
           <p>
             In the context of this vocabulary, we consider the specifics of
             workflows out of scope. We acknowledge that different use cases may
             have complicated workflows, which may yield many individual
             presentations of credentials.
           </p>
+          
+          <p class="note">
+            There are several systems for workflow definitions. See 
+            <a href="https://www.omg.org/spec/BPMN/2.0/About-BPMN/" target="_blank">BPMN</a>,
+            <a href="https://en.wikipedia.org/wiki/YAWL" target="_blank">YAWL</a>, and/or
+            <a href="https://docs.github.com/en/actions/using-workflows/about-workflows" target="_blank">GitHub Workflows</a>.
```

Comment:
does Github Workflows fit here? 
This section describes Supply chain workflows

Reply:
it's just an example... so that folks who are less familiar with workflows can read more about what they are.